### DISCUSSION TITLE
osu lazer crashes on startup  when importing rulesets from itunes

### DISCUSSION THREAD

**Discussion by rafroad** [ppy/osu#16810] [Q&A] [answered]
i was trying to import rulesets through itunes by replacing the rulesets folder but when i started osu it crashes on startup are there any other way to import rulesets and collections to lazer on ios? i have included the logs as well
[osu.iOS-2022-02-07-144429.zip](https://github.com/ppy/osu/files/8013302/osu.iOS-2022-02-07-144429.zip)

**Comment by peppy** [comment, marked as answer]
You can't. iOS can not support this.
